common/enablers/enableEpochsHandler.go

```diff
@@ -39,6 +41,330 @@ func (handler *enableEpochsHandler) EpochConfirmed(epoch uint32, _ uint64) {
+	case core.MetaProtectionFlag:
+		return epoch >= handler.enableEpochsConfig.MetaProtectionEnableEpoch
+	case core.AheadOfTimeGasUsageFlag:
+		return epoch >= handler.enableEpochsConfig.AheadOfTimeGasUsageEnableEpoch
+	case core.GasPriceModifierFlag:
+		return epoch >= handler.enableEpochsConfig.GasPriceModifierEnableEpoch
+	case core.RepairCallbackFlag:
+		return epoch >= handler.enableEpochsConfig.RepairCallbackEnableEpoch
+	case core.ReturnDataToLastTransferFlagAfterEpoch:
+		return epoch > handler.enableEpochsConfig.ReturnDataToLastTransferEnableEpoch
+	case core.SenderInOutTransferFlag:
+		return epoch >= handler.enableEpochsConfig.SenderInOutTransferEnableEpoch
+	case core.StakeFlag:
+		return epoch >= handler.enableEpochsConfig.StakeEnableEpoch
+	case core.StakingV2Flag:
+		return epoch >= handler.enableEpochsConfig.StakingV2EnableEpoch
+	case core.StakingV2OwnerFlagInSpecificEpochOnly:
+		return epoch == handler.enableEpochsConfig.StakingV2EnableEpoch
+	case core.StakingV2FlagAfterEpoch:
+		return epoch > handler.enableEpochsConfig.StakingV2EnableEpoch
+	case core.DoubleKeyProtectionFlag:
+		return epoch >= handler.enableEpochsConfig.DoubleKeyProtectionEnableEpoch
+	case core.ESDTFlag:
+		return epoch >= handler.enableEpochsConfig.ESDTEnableEpoch
+	case core.ESDTFlagInSpecificEpochOnly:
+		return epoch == handler.enableEpochsConfig.ESDTEnableEpoch
+	case core.GovernanceFlag:
+		return epoch >= handler.enableEpochsConfig.GovernanceEnableEpoch
+	case core.GovernanceFlagInSpecificEpochOnly:
+		return epoch == handler.enableEpochsConfig.GovernanceEnableEpoch
+	case core.DelegationManagerFlag:
+		return epoch >= handler.enableEpochsConfig.DelegationManagerEnableEpoch
+	case core.DelegationSmartContractFlag:
+		return epoch >= handler.enableEpochsConfig.DelegationSmartContractEnableEpoch
+	case core.DelegationSmartContractFlagInSpecificEpochOnly:
+		return epoch == handler.enableEpochsConfig.DelegationSmartContractEnableEpoch
+	case core.CorrectLastUnJailedFlagInSpecificEpochOnly:
+		return epoch == handler.enableEpochsConfig.CorrectLastUnjailedEnableEpoch
+	case core.CorrectLastUnJailedFlag:
+		return epoch >= handler.enableEpochsConfig.CorrectLastUnjailedEnableEpoch
+	case core.RelayedTransactionsV2Flag:
+		return epoch >= handler.enableEpochsConfig.RelayedTransactionsV2EnableEpoch
+	case core.UnBondTokensV2Flag:
+		return epoch >= handler.enableEpochsConfig.UnbondTokensV2EnableEpoch
+	case core.SaveJailedAlwaysFlag:
+		return epoch >= handler.enableEpochsConfig.SaveJailedAlwaysEnableEpoch
+	case core.ReDelegateBelowMinCheckFlag:
+		return epoch >= handler.enableEpochsConfig.ReDelegateBelowMinCheckEnableEpoch
+	case core.ValidatorToDelegationFlag:
+		return epoch >= handler.enableEpochsConfig.ValidatorToDelegationEnableEpoch
+	case core.IncrementSCRNonceInMultiTransferFlag:
+		return epoch >= handler.enableEpochsConfig.IncrementSCRNonceInMultiTransferEnableEpoch
+	case core.ESDTMultiTransferFlag,
+		core.ESDTNFTImprovementV1Flag:
```

Comment:
what is `ESDTNFTImprovementV1Flag` ? 
In the original code I do not see 2 flags related to the same `ESDTMultiTransferEnableEpoch`. Please re-check

Reply:
I remember we had a discussion for these methods that rely on the same flag, to better create a new method with an appropriate naming, rather than reuse the same method

This led to the following methods, which are marked as duplicate:
https://github.com/multiversx/mx-chain-go/blob/rc/v1.6.0/common/enablers/epochFlags.go#L634

With this new approach, I had to create a new flag for them

Reply:
ok